doc/user-manual/tools/generating-html.rst

```diff
@@ -15,14 +15,31 @@ You can change the way in which the code is highlighted by providing
 your own CSS file instead of the :download:`default, included one
 <../../../src/data/Agda.css>` (use the ``--css`` option).
 
+If you're using Literate Agda with markdown or reStructedText and you
+want to highlight your Agda codes with Agda's HTML backend and render
+the rest of the content (let's call it "literate" part for convenience)
+with some another renderer, you can use the ``--html-highlight=code``
+option, which makes the Agda compiler:
+
+- not wrapping the literate part into ``<a class="Background">`` tags
+- not wrapping the generated document with a ``<html>`` tag,
+  which means you'll have to specify the CSS location somewhere else,
+  like ``<link rel="stylesheet" type="text/css" href="Agda.css">``
+- converting paired ``<a class="Markup">`` tags into
+  ``<pre class="agda-code">`` tags that wrap the complete Agda code
+  block below
+
 Options
 -------
 
-:samp:`--html-dir {directory}`
+:samp:`--html-dir={directory}`
   Changes the directory where the output is placed to
   :samp:`{directory}`. Default: ``html``.
 
-:samp:`--css {URL}`
+:samp:`--css={URL}`
   The CSS_ file used by the HTML files (:samp:`{URL}` can be relative).
 
+:samp:`--html-highlight=[code,all]`
+  Highlight Agda code only or everything in the generated HTML files.
+
```

Comment:
Please document what the default is here.

Reply:
Ok, I'll fix that.